package.json

```diff
@@ -1,19 +1,22 @@
 {
   "name": "contentful-sdk-core",
-  "version": "1.0.0-determined-by-semantic-release",
+  "version": "3.5.7",
```

Comment:
this we can bring it back to `0.0.0-placeholder` since we finished the manual release